packages/material-ui-lab/src/ToggleButton/ToggleButton.js

```diff
@@ -85,39 +95,28 @@ class ToggleButton extends React.Component {
     }
   };
 
-  render() {
-    const {
-      children,
-      classes,
-      className,
-      disabled,
-      disableFocusRipple,
-      innerRef,
-      selected,
-      ...other
-    } = this.props;
-
-    return (
-      <ButtonBase
-        className={clsx(
-          classes.root,
-          {
-            [classes.disabled]: disabled,
-            [classes.selected]: selected,
-          },
-          className,
-        )}
-        disabled={disabled}
-        focusRipple={!disableFocusRipple}
-        ref={innerRef}
-        onClick={this.handleChange}
-        {...other}
-      >
-        <span className={classes.label}>{children}</span>
-      </ButtonBase>
-    );
-  }
-}
+  return (
+    <ButtonBase
+      className={clsx(
+        classes.root,
+        {
+          [classes.disabled]: disabled,
+          [classes.selected]: selected,
+        },
+        className,
+      )}
+      disabled={disabled}
+      focusRipple={!disableFocusRipple}
+      innerRef={ref}
```

Comment:
Looks like innerRef still needs to be used here

Reply:
What prevents you from using `ref`?

Reply:
The testRef test fails

Reply:
The test passes after https://github.com/mui-org/material-ui/pull/15017 so I've swapped it to use ref